tests/integration/abi_roundtrip_test.py

```diff
@@ -115,13 +131,15 @@ def roundtrip_setup(abi_type):
     if isinstance(abi_type, tuple):
         abi_type, dynamic_length = abi_type
 
-    return (
-        abi_type,
-        str(abi.type_spec_from_annotation(abi_type)),
-        dynamic_length,
-        ABIRoundtrip(abi.make(abi_type), length=dynamic_length).pytealer(),
+    abi_type_str = str(abi.type_spec_from_annotation(abi_type))
+    roundtrip_or_msg = (
+        "ineligible for round trip test"
+        if abi_type_str in BAD_TYPES
+        else ABIRoundtrip(abi.make(abi_type), length=dynamic_length).pytealer()
     )
 
+    return (abi_type, abi_type_str, dynamic_length, roundtrip_or_msg)
```

Comment:
```suggestion
    roundtrip_or_none = None
    if abi_type_str not in BAD_TYPES:
        roundtrip_or_none = ABIRoundtrip(abi.make(abi_type), length=dynamic_length).pytealer()

    return (abi_type, abi_type_str, dynamic_length, roundtrip_or_none)
```

here's my suggestion reflecting @ahangsu 's remark